src/Query/QueryBuilder.php

```diff
@@ -371,7 +372,7 @@ public function getSQL()
      *
      * @param int|string           $key   Parameter position or name
      * @param mixed                $value Parameter value
-     * @param int|string|Type|null $type  One of the {@link ParameterType} constants or DBAL type
+     * @param int|string|Type|null $type  One of the {@link ParameterType} constants, {@link Types} constants or DBAL type instance
```

Comment:
There are dozens of places in the codebase where this union type is used. Is it the only place where this change is needed?

Reply:
In the other places `ParameterType` class is not mentioned :)

We can use the same full description in all places, if we need to suggest it to user always.

Should we?

Reply:
I'd rather remove the details from here.